### PR TITLE
[PassKit] Update to iOS 10.3 beta 1

### PR DESCRIPTION
**Pull request by dalexsoto**

### CODE REVIEW

File: src/passkit.cs [hunk 1]
```diff
@@ -656,6 +658,10 @@ interface PKPaymentNetwork {
 		[Field ("PKPaymentNetworkAmex")]
 		NSString Amex { get; }
 
+		[iOS (10,3), Watch (3,3)]
```
Comment (VincentDondain) [inline src/passkit.cs]: It would be `Watch (3,2)`, watchOS 3.3 does not exist yet (;

File: src/passkit.cs [hunk 2]
```diff
@@ -687,6 +693,14 @@ interface PKPaymentNetwork {
 		[Watch (3,1), iOS (10,1)]
 		[Field ("PKPaymentNetworkSuica")]
 		NSString Suica { get; }
+
+		[iOS (10,3), Watch (3,3)]
```
Comment (VincentDondain) [inline src/passkit.cs]: Same `Watch (3,2)`.

File: src/passkit.cs [hunk 3]
```diff
@@ -687,6 +693,14 @@ interface PKPaymentNetwork {
...
+		[Field ("PKPaymentNetworkQuicPay")]
+		NSString QuicPay { get; }
+
+		[iOS (10,3), Watch (3,3)]
```
Comment (VincentDondain) [inline src/passkit.cs]: Same `Watch (3,2)`.

File: src/passkit.cs [hunk 4]
```diff
@@ -42,6 +42,8 @@ interface PKContact : NSSecureCoding
 #endif // XAMCORE_2_0
 
 		[iOS (9,2)]
+		[Deprecated (PlatformName.iOS, 10,3, message:"Use SubLocality and SubAdministrativeArea on PostalAddress instead")]
+		[Deprecated (PlatformName.WatchOS, 3,3, message:"Use SubLocality and SubAdministrativeArea on PostalAddress instead")]
```
Comment (VincentDondain) [inline src/passkit.cs]: Same `Watch (3,2)`.

Comment (dalexsoto) [inline src/passkit.cs]: Oh apple headers state `__WATCHOS_AVAILABLE(3.3);` heh thanks Apple...

**Comment by dalexsoto** [comment]
build

**Comment by dalexsoto** [comment]
monotouch-test - https://bugzilla.xamarin.com/show_bug.cgi?id=52162
fsharp / watchOS - https://bugzilla.xamarin.com/show_bug.cgi?id=46517

**Pull request merged**